tests/test_writing_api.py

```diff
@@ -199,4 +200,29 @@ def test_update_judgment_fail(mocked_client):
         "is-a-judgment/uri", b"<judgment></judgment>", ""
     )
     assert response.status_code == 409
-    assert "request needed a checkout first" in response.text
+    assert (
+        response.text
+        == '{"detail":"The resource is not checked out by anyone, but that request needed a checkout first."}'
+    )
+
+
+@patch("openapi_server.apis.writing_api.client_for_basic_auth")
+def test_judgment_validation_fail(mocked_client):
```

Comment:
[suggestion]: could be worth editing the function name to something more specific or adding a description of the test. I think I understand it now but took some time for me due to the mocking in the setup and my limited familiarity with the code.

something like `test_judgment_update_errors_when_marklogic_validation_fails`

or something like

"""
Given an openapi client which raises `MarkLogicValidationFailedError` when saving locked judgement xmls
when a PATCH requests is made to the judgement endpoint
Then the response has status 422
And the detail is equal to the message
""""

the given part is a bit tricky due to the mocking but the best I could think of

Reply:
Gone for 
```
def test_validation_error_message_in_api_response(mocked_client):
    """
    Given that a `MarkLogicValidationFailedError` will be raised when attempting to save the judgment,
    When an updated judgment is sent to the /judgment URL,
    Then the Privileged API response has status 422
        and contains the original MarkLogic XML error message
    """
    ```